jerry-core/debugger/jerry-debugger.c

```diff
@@ -0,0 +1,98 @@
+/* Copyright 2016 University of Szeged.
```

Comment:
1. I suppose the "server side" stuff should goes in a separate commit/merge request
2. Commit message should be more specific, for example Add jerry-debugger.py and Implement socket communication for remote (?) debugger